File: LLM-Translations/GPT-4/codenet/Java/Python/s193347311.py
import sys

def main():
    input_data = sys.stdin.readlines()
    int_loop = int(input_data[0].strip())
    str_taro = ""
    str_hana = ""
    int_taro = 0
    int_hana = 0

    for i in range(1, int_loop * 2, 2):
        str_taro = input_data[i].strip()
        str_hana = input_data[i + 1].strip()
        int_judge = (str_taro > str_hana) - (str_taro < str_hana)
        if int_judge < 0:
            int_hana += 3
        elif int_judge == 0:
            int_taro += 1
            int_hana += 1
        elif 0 < int_judge:
            int_taro += 3

    print(int_taro, int_hana)

if __name__ == '__main__':
    main()

File: LLM-Translations/GPT-4/codenet/Java/C/s193347311.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main() {
    int intLoop;
    scanf("%d", &intLoop);
    char strTaro[100];
    char strHana[100];
    int intTaro = 0;
    int intHana = 0;

    for(int i = 0; i < intLoop; i++){
        scanf("%s", strTaro);
        scanf("%s", strHana);
        int intJudge = strcmp(strTaro, strHana);
        if(intJudge < 0){
            intHana += 3;
        }else if(intJudge == 0){
            intTaro++;
            intHana++;
        }else if(0 < intJudge){
            intTaro += 3;
        }
    }
    printf("%d %d\n", intTaro, intHana);
    return 0;
}
// 
